lib/tasks/superbolt.rake

```diff
@@ -1,7 +1,13 @@
 # assumes an environment task that sets up your app environment
 desc "worker that reads the queue and sends messages through the router as configured"
 task :superbolt => :environment do
-  Superbolt::App.new(Superbolt.app_name, {}).run do |message, logger|
-    Superbolt::Router.new(message, logger).perform
+  begin
+    Superbolt::App.new(Superbolt.app_name, {}).run do |message, logger|
+      Superbolt::Router.new(message, logger).perform
+    end
+  # Rescue SignalException so that we don't spam error handlers like Airbrake
+  # with SIGTERM errors every time a task is killed (Heroku dyno restarts, etc.)
+  rescue SignalException => e
```

Comment:
I hope it's alright to chime in here:

I've been watching these `SignalException` PRs. Assuming I understand `Router#perform`, (that it's a blocking processing loop which breaks and then exits shortly after this exception occurs), they look good from the AWS side. We will still have SIGTERMs for post-deploy restarts and scale downs, but in general not outside of those cases.

(sidenote: SIGKILL causes immediate termination and can't be handled by a user process, so ruby never gets a chance to raise an exception from that signal in particular)